src/commands/mocks/spk-config-deployment.yaml

```diff
@@ -0,0 +1,40 @@
+infra:
+  terraform_check: true
+  git_check: true
+  helm_check: true
+  az_cli_check: true
+  env_var_check: true
+  terraform: "v0.12.6"
+  helm: "v2.14.3"
+  git: "2.17.1"
+  bedrock:
+    source: "git@github.com:microsoft/bedrock.git"
+    tag:
+    repo_type: "Public"
+    private_repo_key:
+  azure:
+    az_cli:
+      azure_cli: "2.0.70"
+    az_auth:
+      service_principal_id: "client-id"
+      service_principal_secret: "client-secret"
+      subscription_id: "subscription-id"
+      tenant_id: "tenant-id"
+    az_backend:
+      resource_group_name: "StorageAccount-ResourceGroup"
+      storage_account_name: "abcd1234"
+      container_name: "tf-state"
+      key: "prod.terraform.tf-state"
+deployment:
+  storage:
+    account_name: "my_storage_account"
```

Comment:
should we just utilize the mocks file we already have, `mocks/spk-config.yaml`? That way we're not modifying multiple files every time configs change